js/com/chrome/apis/Controller.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @license
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+CLASS({
+  package: 'com.chrome.apis',
+  name: 'Controller',
+  extends: 'foam.browser.u2.BrowserController',
+  requires: [
+    'com.chrome.apis.Experiment',
+    'com.chrome.apis.ExperimentActivation',
+    'com.chrome.apis.Origin',
+    'com.chrome.apis.User',
+    'foam.dao.EasyDAO',
+  ],
+
+  exports: [
+    'activationDAO as ExperimentActivationDAO',
```

Comment:
I believe the exported name should be experimentActivationDAO,

I prefer just naming the property this and exporting it plainly rather than using an as